models/eos/static_routes/eos_static_routes.yml

```diff
@@ -0,0 +1,120 @@
+---
+GENERATOR_VERSION: '1.0'
+
+ANSIBLE_METADATA: | 
+  {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'network'
+  }
+NETWORK_OS: eos
+RESOURCE: static_routes
+COPYRIGHT: Copyright 2019 Red Hat
+
+DOCUMENTATION: |
+  module: eos_static_routes
+  version_added: 2.10
+  short_description: Configures and manages attributes of static routes on Arista EOS platforms.
+  description: This module configures and manages the attributes of static routes on Arista EOS platforms.
+  author: Gomathi Selvi Srinivasan (@GomathiselviS)
+  notes:
+  options:
+    config:
+      description:
+        - A list of configurations for static routes
+      type: list
+      elements: dict
+      suboptions:
+        vrf:
+          description:
+            - The VRF to which the static route(s) belong
+          type: str
+          default: __global__
+        address_families:
+          description: A dictionary specifying the address family to which the static route(s) belong.
+          type: list
+          elements: dict
+          suboptions:
+            afi:
+              description:
+                - Specifies the top level address family indicator.
+              type: str
+              choices: ['ipv4', 'ipv6']
+              required: True
```

Comment:
I think this should be true (lowercase) but check with developers to be sure. Same comment for all the True /False here